src/composables/discord/notifyDiscordSale.ts

```diff
@@ -0,0 +1,112 @@
+import { PNFT } from "../../common/helpers/types";
```

Comment:
triple check issues with using non-relative paths (use @ syntax)

Reply:
also double check error handling + undefined handling

Reply:
See notes here: https://docs.google.com/document/d/1PQnlM9iIYH1uFfvpVAsdDseASceLtLSuk34h1Q16-0c/edit

for handling of undefined with pfnt data sent to discord